lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -175,36 +176,35 @@ def parse_services_response(response)
     end
 
     def parse_service_options_response(response)
-      doc = REXML::Document.new(response)
-      service_node = doc.elements['service']
-      service_code = service_node.get_text("service-code").to_s
-      service_name = service_node.get_text("service-name").to_s
-      options_node = service_node.elements['options']
-      unless options_node.blank?
-        option_nodes = options_node.elements.collect('option') { |node| node }
-        options = option_nodes.inject([]) do |result, node|
-          option = {
-            :code => node.get_text("option-code").to_s,
-            :name => node.get_text("option-name").to_s,
-            :required => node.get_text("mandatory").to_s == "false" ? false : true,
-            :qualifier_required => node.get_text("qualifier-required").to_s == "false" ? false : true
-          }
-          option[:qualifier_max] = node.get_text("qualifier-max").to_s.to_i if node.get_text("qualifier-max")
-          result << option
-          result
-        end
+      doc = Nokogiri.XML(response)
+      doc.remove_namespaces!
```

Comment:
Yeah, I was lazy; I didn't want to include the namespace in every xpath query. :)